readthedocs/templates/projects/domain_form.html

```diff
@@ -27,6 +27,9 @@
       <strong>{% trans 'SSL certificate status' %}: </strong>
       <span>{{ domain.domainssl.status }}</span>
     </p>
+    {% if domain.domainssl.status == 'pending_validation' %}
+      <p class="help_text"><small>{% trans 'Did you setup a CNAME record in DNS pointing at "readthedocs.io"?' %}</small></p>
```

Comment:
Shouldn't `readthedocs.io` get from `deafult_domain` too?

Doesn't look so importatn anyway.

Reply:
The CNAME has to be explicitly this value or Cloudflare won't issue the SSL certificate. For deployments other than readthedocs.org, this code path will never be taken because `domain.domainssl` is in a private repository.